features/google/src/components/importExplanationExpandable/importExplanationExpandable.jsx

```diff
@@ -0,0 +1,329 @@
+import React, { useContext, useState } from "react";
+// import InfoBox from "../infoBox/infoBox.jsx";
+import {
+    PolyButton,
+    PolyImportContext,
+    RoutingWrapper,
+} from "@polypoly-eu/poly-look";
+import { GoogleContext } from "../../context/google-context.jsx";
+import { useHistory } from "react-router-dom";
+import { FileSelectionError, FileImportError } from "@polypoly-eu/poly-import";
+// import ScrollButton from "../buttons/scrollButton/scrollButton.jsx";
+// import scrollSmoothly from "../../utils/smoothScroll.js";
+import "./importExplanationExpandable.css";
+
+const isSectionOpened = (section, importStatus, importSteps) => {
+    return {
+        request:
+            section == importStatus || importStatus == importSteps.beginning,
+        download: section == importStatus,
+        import: section == importStatus,
+        explore: section == importStatus,
+    }[section];
```

Comment:
This is going to be null in case section is not one of those keys. I guess that's OK, since null is falsish, right?

Reply:
Exactly - with one tiny difference, it's not gonna be `null` but `undefined`. Same effect though.